server/src/client_api.rs

```diff
@@ -8,8 +8,19 @@ use unleash_types::client_metrics::{
     from_bucket_app_name_and_env, ClientApplication, ClientMetrics,
 };
 
+#[utoipa::path(
+    path = "/api/client/features",
+    responses(
+        (status = 200, description = "Return feature toggles for api key used that evaluated to true", body = ClientFeatures),
```

Comment:
Maybe we should use `token` instead? Sentence also read a bit weird to me. I was also under the impression that `client/features` returned all features, not only the ones that evaluated to true.

```suggestion
        (status = 200, description = "Return feature toggles for this token", body = ClientFeatures),
```